entrypoint.sh

```
@@ -11,62 +9,62 @@ if [[ "${DB_SERVER:-}" ]]; then
+    fi
+  else
+    return 1  # DB does not exist
+  fi
+}
 
-    # Run update.php to install Wikibase
-    php /var/www/html/w/maintenance/update.php --quick
+if ! db_exists && [ "${INITIALIZE_DB:-false}" = "true" ]; then
+  # Test if required environment variables have been set
+  REQUIRED_VARIABLES=(MW_ADMIN_NAME MW_ADMIN_PASS MW_ADMIN_EMAIL MW_WG_SECRET_KEY DB_SERVER DB_USER DB_PASS DB_NAME)
+  for i in ${REQUIRED_VARIABLES[@]}; do
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)